modules/crowdx/src/lib/store.ts

```diff
@@ -0,0 +1,159 @@
+import { Computed, IObservable, Kind } from './index'
+import Core from '../core'
+
+const reservedKeywords = [
+  '__crowdx_values__',
+  '__crowdx_deriveds__',
+  '__crowdx_observed__',
+  'constructor',
+  '__crowdx_assignProperty__',
+  '__crowdx_assignDerived__',
+  '__crowdx_sendChanges__',
+  'onBecomeObserved',
+  'onBecomeUnobserved',
+  'get'
+]
+
+interface Derived<ValueT = any> {
+  __crowdx_kind__: Kind.Derived
```

Comment:
I am confused on why you added a kind this. Because how it is used here it is a type, not a value. So this will only be visible in typescript.

Reply:
I should remove the Kinds all together and just check by types. Probably a neater solution

Reply:
Yeah I agree.

Reply:
Fixed